releases/v19.1.5.md

```diff
@@ -0,0 +1,74 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.1.5.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach:v19.1.5
+~~~
+
+### SQL language changes
+
+- Add `check_constraints` table to the `information_schema`. [#39688][#39688]
+
+### Bug fixes
+
+- Unary negatives in constant arithmetic expressions are no longer ignored. [#39367][#39367]
+- Propagate [zone configuration](../v19.1/configure-replication-zones.html) to non-gossiped system tables. [#39691][#39691]
+- Prevent unlimited memory usage during SQL range deletions. [#39733][#39733]
+- A crash caused by the presence of [window functions](../v19.1/window-functions.html) in the source of the [`CREATE TABLE AS`](../v19.1/create-table-as.html) statement is fixed. [#40430][#40430]
+- Fix a planning error that could occur when a [common table expression](../v19.1/common-table-expressions.html) with an [`ORDER BY`](../v19.1/query-order.html) was used inside of a [subquery](../v19.1/subqueries.html). [#40490][#40490]
+- Fixed an optimizer panic when building array access expressions. [#40513][#40513]
+- Fix bug where an MVCC value at a future timestamp is returned after a transaction restart. [#40611][#40611]
+- Consider intents in a read's uncertainty interval to be uncertain just as if they were committed values. This removes the potential for stale reads when a causally dependent transaction runs into the not-yet resolved intents from a causal ancestor. [#40611][#40611]
+- Prevent problems on mixed-version 19.1 clusters that are also performing a lookup join on a table that has an ongoing index backfill. [#40739][#40739]
+- The `cockroach` [CLI client commands](../v19.1/cockroach-commands.html) are now able to connect to a server via the environment variable `COCKROACH_URL`. [#40848][#40848]
+- Fix a crash in apply joins. [#40829][#40829]
+
+### Contributors
+
+This release includes 14 merged PRs by 10 authors.
+
+[#39367]: https://github.com/cockroachdb/cockroach/pull/39367
+[#39688]: https://github.com/cockroachdb/cockroach/pull/39688
+[#39691]: https://github.com/cockroachdb/cockroach/pull/39691
+[#39733]: https://github.com/cockroachdb/cockroach/pull/39733
+[#40430]: https://github.com/cockroachdb/cockroach/pull/40430
+[#40490]: https://github.com/cockroachdb/cockroach/pull/40490
+[#40513]: https://github.com/cockroachdb/cockroach/pull/40513
+[#40611]: https://github.com/cockroachdb/cockroach/pull/40611
+[#40739]: https://github.com/cockroachdb/cockroach/pull/40739
+[#40829]: https://github.com/cockroachdb/cockroach/pull/40829
+[#40848]: https://github.com/cockroachdb/cockroach/pull/40848
+[#unknown]: https://github.com/cockroachdb/cockroach/pull/unknown
+[6b14c0aa3]: https://github.com/cockroachdb/cockroach/commit/6b14c0aa3
```

Comment:
This and the above line seems wrong. I think we want to have the note from https://github.com/cockroachdb/cockroach/pull/40942 which is the backport of https://github.com/cockroachdb/cockroach/pull/40941.

```
Release note (bug fix): Detailed crash reports ("panic messages")
could previously be reported in the wrong file, if SQL audit reporting
or statement logging had been activated. This has been corrected and
crash reports will now properly always appear in the main log file.
```

Reply:
Thanks - updating now

Reply:
Should be fixed in latest push.  PTAL once it's green - hopefully the build will be fast :-}